services/core-api/tests/factories.py

```diff
@@ -593,6 +593,7 @@ class Meta:
     permit_no = factory.LazyFunction(RandomPermitNumber)
     permit_status_code = factory.LazyFunction(RandomPermitStatusCode)
     permit_amendments = []
+    deleted_ind = False
```

Comment:
Were there more factory classes that should have this `deleted_ind = False` set? Might not be a big deal, since `False` would be the default value.

Reply:
did just for the transparency in code

Reply:
Should the other factories that had a deleted indicator added to them specify this default too?